curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/63507ebb0c50ce3b9d669cd9.md

```diff
@@ -17,6 +17,8 @@ The `.map()` method is called on the array, and then the `.filter()` method is c
 
 Following that example, remove your `filtered` variable, and chain your `.filter()` call to your `.map()` call above. Do not remove either of the callback functions.
 
+> Note: We are not using `!Number.isNaN()` here because we are chaining the `.filter()` method after the `.map()` method. The `.map()` method will return an array of numbers, so we can use the `!isNaN()` function without the `Number.` prefix.
```

Comment:
I don't think you ever need to prefix the `isNaN` function with `Number.`, so I don't think this note is necessary. It might be better to fix the instructions in step 7 and just remove `Number.` from `isNaN`. I just tested and step 7 will pass without `Number.`.

Reply:
Yea. True. I think this makes more sense. I will adjust step 7 instead since it passes.